packages/create-app.template.react.beta/templates/src/client.ts.hbs

```diff
@@ -40,8 +36,8 @@ export const auth = createPublicOauthClient(
  * Initialize the client to interact with the Ontology SDK
  */
 const client: Client = createClient(
-  url,
-  $ontologyRid,
```

Comment:
It's a bit weird in the single stack case that we add duplication for the ontology rid when this is already part of the SDK but I don't think it's a big deal when we prefill this correctly most of the time